rdr_service/model/genomics.py

```diff
@@ -54,6 +54,31 @@ class GenomicSetMember(Base):
 
     __tablename__ = "genomic_set_member"
 
+    history_table = True
+    exclude_column_names_from_history = [
+        'modified',
+        'reconcile_metrics_bb_manifest_job_run_id',
+        'reconcile_gc_manifest_job_run_id',
+        'reconcile_metrics_sequencing_job_run_id',
+        'reconcile_cvl_job_run_id',
+        'gem_a1_manifest_job_run_id',
+        'gem_a2_manifest_job_run_id',
+        'gem_a3_manifest_job_run_id',
+        'aw3_manifest_job_run_id',
+        'aw4_manifest_job_run_id',
+        'aw2f_manifest_job_run_id',
+        'cvl_w1_manifest_job_run_id',
+        'cvl_w2_manifest_job_run_id',
+        'cvl_w3_manifest_job_run_id',
+        'cvl_w4_manifest_job_run_id',
+        'cvl_w4f_manifest_job_run_id',
+        'cvl_aw1c_manifest_job_run_id',
+        'cvl_aw1cf_manifest_job_run_id',
+        'cvl_w3sr_manifest_job_run_id',
+        'cvl_w2sc_manifest_job_run_id',
+        'color_metrics_job_run_id'
+    ]
```

Comment:
If we want to add a column to this exclusion list after it was added in a previous release, do we need to create a new migration or will adding it to this list be picked up in the `AutoHistoryRevisionGenerator` run?

Reply:
That should be OK to defer for now. Going forward, if we create a migration for a new field and we want it excluded, we'll add it to the list at that time.